package.json

```diff
@@ -42,20 +42,22 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@microsoft/api-extractor-model": "^7.28.14",
-    "@typescript-eslint/eslint-plugin": "^7.7.0",
-    "@typescript-eslint/parser": "^7.7.0",
+    "@microsoft/api-extractor-model": "^7.29.4",
+    "@types/node": "^22.1.0",
+    "@typescript-eslint/eslint-plugin": "^8.0.1",
+    "@typescript-eslint/parser": "^8.0.1",
     "chai": "^4.4.1",
-    "eslint-config-prettier": "^8.10.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "mocha": "^10.4.0",
+    "eslint": "^9.8.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "mocha": "^10.7.0",
     "nyc": "^15.1.0",
-    "prettier": "^2.8.8",
-    "sinon": "^17.0.1",
+    "prettier": "^3.3.3",
+    "sinon": "^18.0.0",
     "sinon-chai": "^3.7.0",
     "source-map-support": "^0.5.21",
     "standard-version": "^9.5.0",
     "ts-node": "^10.9.2",
-    "typescript": "^5.0.4"
```

Comment:
When i run `ncu -i` locally it provides the option to update the legacy driver's mongodb dep:

`mongodb      ^6.0.0  →   ^6.8.0`

I'm curious, why does the legacy driver depend on the main driver? And is there a reason we didn't update it?

Reply:
I'm not sure why we have a hard dependency, @nbbeeken do you know?  I'd expect it to be a peer dependency.

I didn't update it because it's not a dev dependency.

Reply:
> I didn't update it because it's not a dev dependency.

Correct, this should not be updated. 

It is not a peer dependency because that is not the intended dependency flow. The `mongodb-legacy` package is supposed to replace your `mongodb` dependency in imports and package.json. The caret allows for this package to be installed with any 6.x.x version.

Reply:
Thanks for clarifying Neal, the PR LGTM then!